ui/src/app/applications/components/application-sync-options/application-sync-options.tsx

```diff
@@ -94,6 +94,7 @@ const syncOptions: Array<(props: ApplicationSyncOptionProps) => React.ReactNode>
     props => booleanOption('ApplyOutOfSyncOnly', 'Apply Out of Sync Only', false, props, false),
     props => booleanOption('RespectIgnoreDifferences', 'Respect Ignore Differences', false, props, false),
     props => booleanOption('ServerSideApply', 'Server-Side Apply', false, props, false),
+    props => booleanOption('RollingSync', 'Rolling Sync', false, props, false),
```

Comment:
Hmmm, I thought the rolling sync option is currently specified on the application set.  It's not specifically set at the application level, from what I understand.